k8s/aat/common/sscs/sscs-tya-notif.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         JOB_SCHEDULER_DB_USERNAME: notification@sscs-tya-notif-postgres-v11-db-aat
         JOB_SCHEDULER_DB_NAME: notification
         JOB_SCHEDULER_DB_HOST: sscs-tya-notif-postgres-v11-db-aat.postgres.database.azure.com
+        SSCS_TRACK_YOUR_APPEAL_LINK: https://track-appeal.aat.platform.hmcts.net/trackyourappeal/appeal_id
```

Comment:
Is this the correct place to add this change https://github.com/hmcts/sscs-track-your-appeal-notifications/blob/master/charts/sscs-tya-notif/values.yaml#L44 ?

Reply:
Yes I think so as:

- It's equivalent to how the PROD value is used which is important for the test.
- It's a AAT specific value, and not the default value to be injected.

Aside: I think it is right we are doubly vigilant re config changes, from what I can see our BE services will fallback to AAT or other values in PROD if conf not correctly set; where I think it's safer the app fails to start and deployment fails with existing running pods still up.

Reply:
Cool, as long as that change in notifications is not lost.